protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -135,9 +135,9 @@ class FieldInfo<T> {
   /// Returns a read-only default value for a field. Unlike
   /// [GeneratedMessage.getField], doesn't create a repeated field.
   dynamic get readonlyDefault {
-    if (isRepeated) {
-      return _emptyList ??= FrozenPbList._([]);
-    }
+    // if (isRepeated) {
```

Comment:
Same here.

Reply:
I wanted to enable this later. I still don't understand why this line is needed, but decided to not change more than necessary for now.